.github/workflows/makefile.yml

```diff
@@ -0,0 +1,27 @@
+name: Makefile CI
+
+on:
+  push:
+    branches: [ pr-github-actions ]
```

Comment:
[.github/workflows/makefile.yml#5](https://github.com/ivansoft/PortfolioAnalyser/pull/3/files#diff-ab2f63759f09e4a4c7d039b6831fe49d10a7b39c9ea91d86ee5e516c84966003R5):
```suggestion
```
https://docs.github.com/en/actions/security-guides/automatic-token-authentication
> When you enable GitHub Actions, GitHub installs a GitHub App on your repository. 
Once installed - 'pr-github-actions' trigger is not needed any more.

Reply:
suggestion commited - https://github.com/ivansoft/PortfolioAnalyser/pull/3/commits/8f44e594510eb2a2336f273a90f9ad4ae61a965f